homer.root/usr/share/clearwater/infrastructure/scripts/create-homer-nginx-config

```diff
@@ -43,27 +43,23 @@ split_domain_and_port()
 
 . /etc/clearwater/config
 
-if [ -z $xdms_hostname ]
+if [ -n "$xdms_hostname" ]
```

Comment:
A comment about how this protects us in case shared_config is missing would be good.